src/components/CTA.tsx

```diff
@@ -28,10 +27,7 @@ export const CTA = () => (
       as={ChakraLink}
       isExternal
       href="https://github.com/vercel/next.js/blob/canary/examples/with-chakra-ui-typescript"
-      variant="solid"
-      colorScheme="green"
-      rounded="button"
-      flexGrow={3}
+      variant="customVariant"
```

Comment:
i think you should add a more intuitive name for this style instead of a default name

Reply:
Ok, maybe the button could be "fullBlueButton" and the input could be "darkGrayInput"?

Reply:
> i think you should add a more intuitive name for this style instead of a default name

Ok, maybe the button could be "fullBlueButton" and the input could be "darkGrayInput"?